src/app/games/hive/hive.component.html

```diff
@@ -0,0 +1,73 @@
+<svg xmlns="http://www.w3.org/2000/svg"
+     width="100%" height="100%"
+     [attr.viewBox]="viewBox"
+     preserveAspectRatio="xMidYMid meet">
+    <g *ngFor="let groundInfo of ground">
+        <polygon id="space_{{ groundInfo.x }}_{{ groundInfo.y }}"
+                 [attr.points]="getHexaPointsAtXY(groundInfo.x, groundInfo.y)"
+                 class="base mid-stroke"
+                 [ngClass]="groundInfo.content.spaceClasses"
+                 (click)="selectSpace(groundInfo.x, groundInfo.y)"/>
+    </g>
+    <g *ngFor="let groundInfo of ground">
+        <polygon *ngIf="groundInfo.content.strokeClasses.length > 0"
+                 id="stroke_{{ groundInfo.x }}_{{ groundInfo.y }}"
+                 [attr.points]="getHexaPointsAtXY(groundInfo.x, groundInfo.y)"
+                 class="base no-fill small-stroke"
+                 [ngClass]="groundInfo.content.strokeClasses"
+                 (click)="selectSpace(groundInfo.x, groundInfo.y)"/>
+    </g>
+    <g *ngFor="let level of pieces; index as z">
+        <g *ngFor="let pieceInfo of level">
```

Comment:
let pieceInfos of pieceInfosByLevel pour la ligne plus haut :p ?
puis let pieceInfo of pieceInfos ?
Oublie pas de porteur deux chaussettes non-matchant en soutient à moi :D !

Reply:
on est passé sur layer !